LEAF_Request_Portal/sources/Service.php

```diff
@@ -144,19 +144,29 @@ public function removeSyncService(int $groupID): bool
         return true;
     }
 
+    /**
+     * @param int $groupID
+     * @param string $member
+     *
+     * @return bool
+     *
+     * Created at: 8/16/2023, 8:45:10 AM (America/New_York)
+     */
     public function addMember($groupID, $member)
```

Comment:
type hinting and return type

Reply:
I didn't add it here because I'm afraid it will break things. I did add the PHPDoc section to indicate what should be passed in.